src/pages/update/Update.tsx

```diff
@@ -28,6 +33,16 @@ const Update = (props: Props) => {
     return unsubscribe
   }, [props.match.params.id])
 
+  useEffect(() => {
```

Comment:
There's no need to run this in an effect. You can simply run it at every render, kinda like you do with a loading message.

Putting it in an effect you risk to have a short flash between the first render and the time the effect runs.

Reply:
Oh, I'll change it then :+1:

Reply:
Fixed.